rdmd.d

```diff
@@ -717,14 +717,14 @@ private string[string] getDependencies(string rootModule, string workDir,
 }
 
 // Is any file newer than the given file?
-bool anyNewerThan(in string[] files, in string file)
+bool anyNewerThan(T)(T files, in string file)
 {
     yap("stat ", file);
     return files.anyNewerThan(file.timeLastModified);
 }
 
 // Is any file newer than the given file?
-bool anyNewerThan(in string[] files, SysTime t)
+bool anyNewerThan(T)(T files, SysTime t)
 {
     // Experimental: running newerThan in separate threads, one per file
     if (false)
```

Comment:
Btw it seems that this experiment is "over" and parallel compilation is now enabled by default. Should we remove the old cold?